frontend/packages/console-shared/src/constants/common.ts

```diff
@@ -99,3 +99,8 @@ export enum REQUESTER_FILTER {
   USER = 'user',
   SYSTEM = 'system',
 }
+
+export const CLUSTER_VERSION_DEFAULT_UPSTREAM_SERVER_URL_PLACEHOLDER =
+  'https://api.openshift.com/api/upgrades_info/v1/graph';
+export const CLUSTER_VERSION_CUSTOM_UPSTREAM_SERVER_URL_PLACEHOLDER =
+  'https://example.com/api/upgrades_info/v1/graph';
```

Comment:
I should have made this comment in the previous review, but since this is specific to a single form and not really a constant we'd reuse, I'd inline it there.